features/conversation-list/hooks/use-conversations-count.ts

```diff
@@ -0,0 +1,26 @@
+import { createConversationListQueryObserver } from "@/queries/useConversationListQuery";
+import { useCurrentAccount } from "@data/store/accountsStore";
+import { useEffect, useState } from "react";
+
+export const useConversationsCount = () => {
+  const account = useCurrentAccount();
+
+  const [count, setCount] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const unsubscribe = createConversationListQueryObserver({
```

Comment:
Let's come back to this https://github.com/ephemeraHQ/converse-app/pull/1514#discussion_r1915556993